CondFormats/RPCObjects/interface/RPCAMCLink.icc

```diff
@@ -47,134 +47,134 @@ inline int RPCAMCLink::getAMCInput() const
-        if (_value > min_amcinput_)
-            return setAMCInput(_value - 1);
+    int value(0);
+    if ((value = getAMCInput()) != wildcard_) {
+        if (value > min_amcinput_)
+            return setAMCInput(value - 1);
         setAMCInput(max_amcinput_);
     }
-    if ((_value = getAMCNumber()) != wildcard_) {
-        if (_value > min_amcnumber_)
-            return setAMCNumber(_value - 1);
+    if ((value = getAMCNumber()) != wildcard_) {
+        if (value > min_amcnumber_)
+            return setAMCNumber(value - 1);
         setAMCNumber(max_amcnumber_);
     }
-    if ((_value = getFED()) != wildcard_) {
-        if (_value > min_fed_)
-            return setFED(_value - 1);
+    if ((value = getFED()) != wildcard_) {
+        if (value > min_fed_)
+            return setFED(value - 1);
         setFED(max_fed_);
     }
     return *this;
 }
 
 inline RPCAMCLink RPCAMCLink::operator--(int)
 {
-    RPCAMCLink _return(*this);
+    RPCAMCLink _value(*this);
     --(*this);
-    return _return;
+    return _value;
 }
 
-inline int RPCAMCLink::bf_get(int const _min, ::uint32_t const _mask, int const _pos) const
+inline int RPCAMCLink::bf_get(int const min, ::uint32_t const mask, int const pos) const
 {
-    ::uint32_t _value(id_ & _mask);
-    if (_value == 0)
+    ::uint32_t value(id_ & mask);
+    if (value == 0)
         return wildcard_;
-    return _min + (int)(_value >> _pos) - 1;
+    return min + (int)(value >> pos) - 1;
 }
 
-inline RPCAMCLink & RPCAMCLink::bf_set(int const _min, int const _max, ::uint32_t const _mask, int const _pos, int const _value)
+inline RPCAMCLink & RPCAMCLink::bf_set(int const min, int const max, ::uint32_t const mask, int const pos, int const value)
 {
-    if (_value >= _min && _value <= _max) {
-        id_ &= ~_mask;
-        id_ |= (((::uint32_t)(_value - _min + 1) << _pos) & _mask);
-    } else if (_value == wildcard_)
-        id_ &= ~_mask;
+    if (value >= min && value <= max) {
+        id_ &= ~mask;
+        id_ |= (((::uint32_t)(value - min + 1) << pos) & mask);
+    } else if (value == wildcard_)
+        id_ &= ~mask;
     else
         throw cms::Exception("OutOfRange")
-            << "Out-of-range input for RPCAMCLink::bf_set, position " << _pos << ": " << _value;
+            << "Out-of-range input for RPCAMCLink::bf_set, position " << pos << ": " << value;
     return *this;
 }
 
-inline std::ostream &  RPCAMCLink::bf_stream(std::ostream & _ostream, int const _min, ::uint32_t const _mask, int const _pos) const
+inline std::ostream &  RPCAMCLink::bf_stream(std::ostream & ostream, int const min, ::uint32_t const mask, int const pos) const
 {
-    ::uint32_t _value(id_ & _mask);
-    if (_value == 0)
-        return (_ostream << '*');
-    return (_ostream << (_min + (int)(_value >> _pos) - 1));
+    ::uint32_t value(id_ & mask);
+    if (value == 0)
+        return (ostream << '*');
+    return (ostream << (min + (int)(value >> pos) - 1));
```

Comment:
just out of curiosity, was there a specific reason to rename the variables without underscore character?